tests/legacy-cli/e2e/tests/build/server-rendering/server-routes-output-mode-server-platform-neutral.ts

```diff
@@ -69,10 +69,6 @@ export default async function () {
         },
       ];
     `,
-    'src/app/app.config.ts': `
-      import { provideAnimationsAsync } from '@angular/platform-browser/animations/async';
```

Comment:
I dropped this from the test since it seemed irrelevant. Another option was to install animations in the test, but I was having a hard time installing a version that would satisfy the peer dependencies.